src/profile/reducer.js

```
@@ -1,6 +1,18 @@
 import * as constants from 'src/profile/constants';
 import * as statuses from 'src/profile/statuses';
 
+export const getProfile = (state, profileId) => {
+  if (!profileId) return {};
```

If this case were to happen, it'd likely be a bug. I think it'd be better to just throw an error. At a later stage, we can try catch the error (see what I did there ;), and use it to display something to the user that there is an error on our side.